infra/capi/cluster-capi-infrastructure-metal3/README.md

```diff
@@ -0,0 +1,11 @@
+# cluster-capi-infrastructure-metal3
+
+## Description
+
+cluster-api metal3 provider package
+
+## build
+
+```
+clusterctl generate provider --infrastructure metal3:v1.9.0 > cluster-api-infrastructure-metal3.yaml
```

Comment:
Should this be as follows?
```suggestion
clusterctl generate provider --infrastructure metal3:v1.9.0 > cluster-capi-infrastructure-metal3.yaml
```

Reply:
Yeah I was asking about the naming, as you have a file called `cluster-capi-infrastructure-metal3.yaml` committed